Makefile

```
@@ -937,12 +948,25 @@ set-manifest-image:
 clean: ## Remove all generated files
 	$(MAKE) clean-bin
 	$(MAKE) clean-book
+	$(MAKE) clean-tilt
+	$(MAKE) clean-charts
 
 .PHONY: clean-bin
 clean-bin: ## Remove all generated binaries
 	rm -rf $(BIN_DIR)
 	rm -rf $(TOOLS_BIN_DIR)
 
+.PHONY: clean-tilt
+clean-tilt: ## Remove all files generated by Tilt
+	rm -rf ./tiltbuild
+	rm -rf ./controlplane/kubeadm/tiltbuild
```

```suggestion
	rm -rf ./controlplane/kubeadm/.tiltbuild
```

Same for the others.